scripts.d/ta/520_bucket_and_process_uptime.sh

```diff
@@ -36,12 +36,20 @@ CURRENT_TIME_EPOCH=$(                 date +%s)
 if [[ $((${CURRENT_TIME_EPOCH}-${MOST_RECENT_BUCKET_STARTTIME_EPOCH})) -lt 3600 ]]; then
     RETURN_CODE="254"
     echo "Weka buckets have been restarted within the last hour, or have never started. This may not be a problem on a new cluster"
-    echo "but could be indicative of problems (e.g. network flapping"
+    echo "but could be indicative of problems (e.g. network flapping)"
+    echo "Recommended Resolutions:"
+    echo " . If this is a new cluster,  or hosts have been upgraded/reboot, this is likely expected"
+    echo " . Otherwise the most likely cause is network problems, such as link flapping or congestion."
+    echo " . Review hardware and network stability, then contact customer success"
 fi
 if [[ $((${CURRENT_TIME_EPOCH}-${MOST_RECENT_PROCESS_STARTTIME_EPOCH})) -lt 3600 ]]; then
     RETURN_CODE="254"
     echo "Weka processes have been restarted within the last hour, or have never started. This may not be a problem on a new cluster"
     echo "but could be indicative of problems (e.g. network flapping"
+    echo "Recommended Resolutions:"
+    echo " . If this is a new cluster,  or hosts have been upgraded/reboot, this is likely expected"
```

Comment:
Remove spurious space after first comma.

Change upgraded/reboot to upgraded/rebooted.